src/main/java/ticTacToe/game/GameOption.java

```diff
@@ -0,0 +1,29 @@
+package ticTacToe.game;
+
+public enum GameOption {
+
+    HUMAN_VS_HUMAN(1),
+    HUMAN_VS_COMPUTER(2),
+    COMPUTER_VS_COMPUTER(3);
+
+    private final int optionNumber;
+
+    GameOption(int number) {
+        optionNumber = number;
+    }
+
+    public static boolean isValid(int gameOptionNumber) {
+        return gameOptionNumber == HUMAN_VS_HUMAN.optionNumber || gameOptionNumber == HUMAN_VS_COMPUTER.optionNumber
```

Comment:
Might be worth splitting this over three lines (you can hit 'enter' after the `||` signs)